### DISCUSSION TITLE
Ubuntu Server 21.04 & Jamulus

### DISCUSSION THREAD

**Discussion by Helondeth** [jamulussoftware/jamulus#1566] [Technical Support & Help] [answered]
Hello all,
I have updated my Ubuntu Server 20.10 to the new Ubuntu Server 21.04. Unfortunately, I can no longer get Jamulus to run. When I try to start the server I always get the error message "code=exited, status=127".
Can someone tell me what is the reason for this?

After the update from 20.04 to 20.10 I did not have this problem.

Kind regards

**Comment by hoffie** [comment, marked as answer]
Hi,

the error message sounds like one from systemd, is this correct? The error code may indicate that the path is no longer correct.

How did you install Jamulus? What does `systemctl cat jamulus` output? What does `journalctl -u jamulus` show?

**Reply by Helondeth** [reply]
It works now.
I installed jamulus with the headless .deb file again and everything works fine

Thanks for your help
